docs/docsite/rst/user_guide/playbooks_best_practices.rst

```diff
@@ -30,11 +30,11 @@ The top level of the directory would contain files and directories like so::
     staging                   # inventory file for staging environment
 
     group_vars/
-       group1                 # here we assign variables to particular groups
-       group2                 # ""
+       group1.yml             # here we assign variables to particular groups
+       group2.yml
     host_vars/
-       hostname1              # if systems need specific variables, put them here
-       hostname2              # ""
+       hostname1.yml          # here we assign variables to particular system
```

Comment:
Please add an 's' to the end of 'systems' or an 'a' in front of the word 'particular'

Reply:
completed, not sure how to remove needs review.